### ISSUE TITLE
Spatial BatchNormalization for mxnet models

### ISSUE THREAD

**Issue by ashkanaev** [onnx/onnx-tensorrt#149]
Hello! 

How can i handle correct converting of batch normalization Op for mxnet models. By default MXnet BN works in non-spatial model 

`bn_node = onnx.helper.make_node(

        "BatchNormalization"
        input_nodes,
        [name],
        name=name,
        epsilon=eps,
        momentum=momentum,
        # MXNet computes mean and variance per feature for batchnorm
        # Default for onnx is across all spatial features. So disabling the parameter.
        spatial=0
    )`

At the moment spatial flag is deprecated.... 
Please help! 
Thanks!

**Comment by luan1412167** [comment]
@ashkanaev Did you solve it? Can you share to me?

**Comment by oscarriddle** [comment]
@ashkanaev 
Tensorrt use IElementWiseLayer to rawly do the BatchNormalization, so it seems impossible to support the momentum batch normalization.

**Comment by kevinch-nv** [comment]
I'm not too familiar with the MXNet->ONNX conversion process, this may be an issue to open in a different repo such as the MXNet one or main ONNX repo.

**Comment by kevinch-nv** [comment]
Closing due to inactivity - if you are still having issues with the latest version of onnx-tensorrt feel free to open a new issue.
